ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PointSchedule.hs

```diff
@@ -460,6 +460,8 @@ data GenesisTest blk schedule = GenesisTest
     gtLoPBucketParams    :: LoPBucketParams,
     gtCSJParams          :: CSJParams,
     gtSlotLength         :: SlotLength,
+    -- | The number of extra honest peers we want in the test.
+    -- It is stored here for convenience, and because it may affect schedule and block tree generation.
```

Comment:
```suggestion
    -- It is stored here for convenience, and because it may affect schedule and block tree generation.
    --
    -- The will be one adversarial peer per alternative branch, and
    -- `1 + gtExtraHonestPeers` honest peers.
```